utility.py

```diff
@@ -1,4 +1,5 @@
 #External
+from random import paretovariate
```

Comment:
Again probably not intentional import I guess XD

Reply:
I don’t even remember adding these